components/formats-gpl/src/loci/formats/in/SVSReader.java

```diff
@@ -64,16 +71,12 @@
     LoggerFactory.getLogger(SVSReader.class);
 
   /** TIFF image description prefix for Aperio SVS files. */
-  public static final String APERIO_IMAGE_DESCRIPTION_PREFIX = "Aperio Image";
+  private static final String APERIO_IMAGE_DESCRIPTION_PREFIX = "Aperio Image";
```

Comment:
Not that I don't approve of more `private`s, but what's the reasoning behind this change? Do we know there are no consumers?